speed-transition.lua

```diff
@@ -30,6 +49,9 @@ function check_should_speedup()
    local subdelay = mp.get_property_native("sub-delay")
    mp.command("no-osd set sub-visibility no")
    mp.command("no-osd sub-step 1")
+   while mp.get_property("sub-text") and shouldIgnore(mp.get_property("sub-text")) do
+      mp.command("no-osd sub-step 1")
+   end
```

Comment:
if the next sub line contains an ignored character keep searching ahead for one that doesn't. I pretty sure when sub-step doesn't find anything (not enough in cache or EOF) this exits because sub-text is empty.